src/glmtools.jl

```diff
@@ -88,6 +88,24 @@ struct NegativeBinomialLink  <: Link
     θ::Float64
 end
 
+"""
+    PowerLink
+
+A [`Link`](@ref) defined as `η = μ^λ` when `λ ≠ 0`, and to `η = log(μ)` when `λ = 0`.
+
+PowerLink refers to the class of transforms that use a power function or logarithmic function to transform responses into a Gaussian or a Gaussian-like.
```

Comment:
Actually I'm not sure we should even mention this given that you can theoretically combine this link with distributions other than Normal, and we don't mention it for other links which are mathematically equivalent.

Reply:
Removed the last part - Now it is `PowerLink refers to the class of transforms that use a power function or logarithmic function to transform responses.`